### PR TITLE
[1.26.0] identity service vendor docs

### PR DESCRIPTION
**Pull request by salahalsaleh**
Image 2020-12-17 at 8 56 06 AM

Image 2020-12-17 at 8 56 27 AM

Image 2020-12-17 at 8 56 45 AM

### CODE REVIEW

File: content/vendor/identity-service/configuring-identity-service.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+weight: 20020
+---
+
+When enabling the identity service for your application, KOTS will deploy [Dex](https://dexidp.io/) as an intermediary that can be configured to control access to the application.
+Dex implements an array of protocols for querying other user-management systems, known as [connectors](https://dexidp.io/docs/connectors/).
+This feature is only available for licenses that have the Identity Service feature enabled.
+
+![Identity Service License Field](/images/identity-service-license-field.png)
+
+The KOTS Identity custom resource enables and configures the Identity Service for your application. If you prefer, here is an [example application](https://github.com/replicatedhq/kots-idp-example-app) that demonstrates how to configure the Identity Service.
```
Comment (emosbaugh) [inline content/vendor/identity-service/configuring-identity-service.md:14]: newline to separate sentences

File: content/vendor/identity-service/configuring-identity-service.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+To begin, create a new release on the [Vendor portal](https://vendor.replicated.com).
+Once you are editing the release, create a new [KOTS Identity CRD](https://kots.io/vendor/additional-objects/identity/) file customized for your application.
```
Comment (emosbaugh) [inline content/vendor/identity-service/configuring-identity-service.md:17]: can you make these links relative?

```
/vendor/additional-objects/identity/
```

File: content/vendor/identity-service/configuring-identity-service.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+![Identity Service CRD](/images/identity-service-crd.png)
+
+The Identity Service has to be accessible from the browser, for that reason, KOTS provides the service name and port to the app through the [identity template functions](https://kots.io/reference/template-functions/identity-context/) so that the app can then configure ingress for the identity service, for example:
```
Comment (emosbaugh) [inline content/vendor/identity-service/configuring-identity-service.md:21]:
```
/template-functions/identity-context/
```

**Review by emosbaugh — APPROVED** [review]

**Comment by markpundsack** [comment]
These all need Beta labels.

**Comment by markpundsack** [comment]
Should this be under "Licenses and Entitlements"? It certainly should be listed on the "Built-In Entitlements" page.

**Comment by markpundsack** [comment]
The content and flow could be improved here. I'd like to see everything a vendor needs in one place, but currently you need to jump around between this vendor doc, some reference docs, and an example application, before you really know what you need to do. But, at least it's all linked and there.

1. Add Beta tag to Enabling and Configuring Identity Service
2. Add Beta tag to CRD reference
3. Add to Built-in Entitlements
3. Consider moving this under Licenses and Entitlements 

I'm not positive this page has enough information for a vendor to get started.

**Pull request merged**